docs/reference/glossary.md

```diff
@@ -2235,61 +2235,66 @@ Trusted Execution Environment
 
 TLS
 Transport Layer Security
-    *Work in Progress*
+    TLS is a cryptographic protocol that provides secure communication over a network by encrypting data transmitted between clients and servers. It is the successor to SSL (Secure Sockets Layer) and is what makes {term}`HTTPS` secure (the padlock icon in your browser).
 
 tmpfs
 Temporary Filesystem
-    *Work in Progress*
+    tmfs is a filesystem that stores all files in virtual memory (RAM) instead of on a physical disk, making it extremely fast but also volatile - data disappears when the system reboots or loses power.
 
 tmux
 Terminal Multiplexer
-    *Work in Progress*
+    tmux is a powerful command-line tool that allows you to manage multiple terminal sessions from a single window, with the ability to detach and reattach sessions. It is a way to have multiple terminal windows and split-screen layouts, all within a single terminal, with sessions that persist even when you disconnect.
 
 topologies
-    *Work in Progress*
+    Topologies (in a networking context) refer to the physical or logical arrangement of devices and connections in a network - essentially how computers, switches, routers, and other devices are interconnected.
 
 TOTP
 Time-based One-Time Password
-    *Work in Progress*
+    TOTP is an algorithm that generates temporary, single-use passwords based on the current time and a shared secret key. It Is one of the most common forms of two-factor authentication (2FA) used by apps like Google Authenticator, Authy, Microsoft Authenticator, and many others.
 
 TPM
 Trusted Platform Module
     A hardware security chip that securely stores cryptographic keys, measures
     system integrity during boot, and enables features such as secure storage,
     encryption, and attestation.
 
-traceback
-    *Work in Progress*
+Traceback
+    traceback is term used in Python. Traceback is a report of the function call stack printed when an exception occurs, showing the sequence of calls leadiing to the error, including file names, line numbers, and the exception type. It helps developers identify and debug issues by traciing execution backward from the failure point.
 
 Traceroute
-    *Work in Progress*
+    Traceroute is a network diagnostic tool that shows the path packets take from your computer to a destination across the internet, revealing each hop (router) along the way.
 
-ttys
+TTYS
 Teletype Terminals
-    *Work in Progress*
+    TTYS were electromechanical typewriter-like devices used as input/output terminals for telegraph networks and early computers. They sent and received text over serial lines, printing characters on paper instead of displaying them on a screen. Even though the physical machines are mostly gone, the term tty still exists in Unix/Linux systems to represent:
+* A text-based input/output device
+* A terminal session
+* A console connection
```

Comment:
question: Does this need to be indented?

Reply:
This does need to be indented to the same depth as the rest of the definition, otherwise Sphinx won't pick up the bullet points as being part of the definition